tools/tsp-client/src/index.ts

```diff
@@ -1,16 +1,15 @@
-import { npmCommand } from "./npm.js";
+import { npmCommand, npxCommand } from "./npm.js";
```

Comment:
Opened an issue to refactor the code using `npmCommand` and `npxCommand` into one function: https://github.com/Azure/azure-sdk-tools/issues/7893

Will have a follow up PR for that